app/src/main/res/values/strings.xml

```diff
@@ -318,6 +318,14 @@
     <string name="sharedpreference_card_details_show_validity" translatable="false">sharedpreference_card_details_show_validity</string>
     <string name="sharedpreference_card_details_show_archived_cards" translatable="false">sharedpreference_card_details_show_archived_cards</string>
     <string name="settings_category_title_cards">Card view</string>
+    <string name="settings_category_title_cards_overview">Cards overview</string>
+    <string name="settings_column_count">Columns</string>
+    <string name="settings_automatic_column_count">Automatic</string>
+    <string name="settings_key_automatic_column_count" translatable="false">default</string>
+    <string name="settings_column_count_1">1: 💳</string>
+    <string name="settings_column_count_2">2: 💳💳</string>
+    <string name="settings_column_count_3">3: 💳💳💳</string>
```

Comment:
I understand what you were trying to do, but this may hurt screenreader users and I'm pretty sure some translator will break the translation for this. So let's keep it simple :)

```suggestion
    <string name="settings_column_count_1">1</string>
    <string name="settings_column_count_2">2</string>
    <string name="settings_column_count_3">3</string>
```

Reply:
undone